util/topgen/strong_random.py

```diff
@@ -36,14 +34,18 @@ def load(self, input_file):
             log.error("Entropy buffer " + input_file + " can't be loaded twice.")
             sys.exit(1)
 
-        self.buffer = np.loadtxt(input_file)
+        # self.buffer = np.loadtxt(input_file)
+        with open(input_file, 'r') as fp:
+            for line in fp:
+                x = int(line[:-1])
```

Comment:
What does the `[:-1]` remove? If it's a comma, maybe:
```suggestion
                x = int(line.strip(','))
```
And if the values are expected to be bytes, it might be good to add a range check.

Reply:
I thought that `[:-1]` was needed to remove the line breaks, but it works well without it. It's fixed now and I've added the range check.